services/proxy/src/Proxy/API/Internal.hs

```diff
@@ -15,29 +15,19 @@
 -- You should have received a copy of the GNU Affero General Public License along
 -- with this program. If not, see <https://www.gnu.org/licenses/>.
 
-module Proxy.API
-  ( sitemap,
+module Proxy.API.Internal
+  ( InternalAPI,
+    servantSitemap,
   )
 where
 
 import Imports hiding (head)
-import Network.Wai.Predicate (true)
-import Network.Wai.Routing (Routes, continue, get, head)
-import Network.Wai.Utilities (empty)
-import Proxy.API.Public qualified as Public
-import Proxy.Env (Env)
-import Proxy.Proxy (Proxy)
+import Proxy.Proxy qualified
+import Servant
+import Wire.API.Routes.MultiVerb
+import Wire.API.Routes.Named (Named (Named))
 
-sitemap :: Env -> Routes a Proxy ()
-sitemap e = do
-  Public.sitemap e
-  routesInternal
+type InternalAPI = Named "status" ("i" :> "status" :> MultiVerb 'GET '[Servant.JSON] '[RespondEmpty 200 "OK"] ())
```

Comment:
Don't we need a route for the HTTP HEAD requests anymore? Or, is it implicitly generated? :thinking: 

The old version had:
```haskell
routesInternal = do
  head "/i/status" (continue $ const (pure empty)) true
...
```

Reply:
yes, it's implicit.